core/frontend/src/components/vehiclesetup/configuration/lights.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="d-flex">
+    <v-card>
+      <v-card-title> Lights 1 (RCIN9) </v-card-title>
+      <v-card-text>
+        Here you can configure what pin outputs the signal for the first set of lights.
+        <v-select
+          v-model="lights1_new_param"
+          :items="recommended_params"
+          :item-text="friendlyName"
+          :item-value="'name'"
+          label="Lights 1"
+        />
+      </v-card-text>
+    </v-card>
+    <v-card>
+      <v-card-title> Lights 2 (RCIN10) </v-card-title>
+      <v-card-text>
+        What pin outputs the signal for the second set of lights.
+        <v-select
+          v-model="lights2_new_param"
+          :items="recommended_params"
+          :item-text="friendlyName"
+          :item-value="'name'"
+          label="Lights 2"
+        />
+      </v-card-text>
+    </v-card>
+    <v-card>
+      <v-card-title> Joystick steps </v-card-title>
+      <v-card-text>
+        How many button presses it takes to go from 0% to 100% brightness.
+        <br>
+        {{ light_steps?.value }} steps result in a {{ (100 / light_steps?.value).toFixed(1) }}%
+        increase per button press.
+        <v-text-field
+          ref="steps_input"
+          v-model="steps_new_value"
+          label="Joystick steps"
+          type="number"
+          @input="setSteps"
+        />
+      </v-card-text>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+import mavlink2rest from '@/libs/MAVLink2Rest'
+import autopilot_data from '@/store/autopilot'
+import Parameter, { printParam } from '@/types/autopilot/parameter'
+
+enum Lights {
+  Lights1 = 59, // RCIN9
+  Lights2 = 60, // RCIN10
+  DISABLED = 0,
+}
+
+export default {
+  name: 'LightsConfigration',
+  data() {
+    return {
+      lights1_new_param: undefined as (string | undefined), // item-value must be a primitive
+      lights2_new_param: undefined as (string | undefined),
+      steps_new_value: 0,
```

Comment:
could we use a default value, like 10 ?
Other than that is just weird to start with 0 steps.

Reply:
that should never happen, as we set it to the parameter value both when mounting (if the param is available) or when we detect a parameter change in the watcher. but I did update it anyway.

Reply:
When you configure it for the first time, if the lights are not configured, it starts with 0.